conf/application.conf

```diff
@@ -152,6 +152,11 @@ vat-agent-client-lookup-frontend {
   path = "/vat-through-software/agent-lookup/client-vat-number"
 }
 
+manage-vat-subscription-frontend {
+  url = "http://localhost:9150"
+  path = "/vat-through-software/account/change-business-details/non-agent"
```

Comment:
Could this URL be updated to exclude the `non-agent` part from the end? If you hit that route without specifying  a user then their controller dynamically determines whether a user is an agent or not and redirects them.